src/routes/routes.tsx

```diff
@@ -8,74 +8,75 @@ import { Profile } from "../pages/profile/Profile";
 import { MapView } from "../pages/map/MapView";
 import { PlaceController } from "../contexts/PlaceContext";
 import Layout from "../components/layout/Layout";
-import PlaceTable from "../components/places/placeTable/PlaceTable";
+// import PlaceTable from "../components/places/placeTable/PlaceTable";
```

Comment:
Why does placeTable is commented?

Reply:
'PlaceTable' is declared but its value is never read. Do i delete de line?

Reply:
done